.github/workflows/ci.yml

```
@@ -0,0 +1,28 @@
+name: Jest Test
+
+on:
+  push:
+    branches: ["master", "main", "develop", "feat/init-repo"]
+  pull_request:
+    branches: ["master", "main", "develop", "feat/init-repo"]
```

The main branch is enough.

```suggestion
    branches: ["main"]
  pull_request:
    branches: ["main"]
```